src/main/webapp/js/feedbackResponseComments.js

```diff
@@ -366,26 +366,53 @@ function showNewlyAddedResponseCommentEditForm(addedIndex) {
     $("#responseCommentEditForm-" + addedIndex).show();
 }
 
-function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, sender) {
+function loadFeedbackResponseComments(user, courseId, fsName, fsIndx, clickedElement) {
     $(".tooltip").hide();
-    var panelBody = $(sender).parent().find('div[class^="panel-body"]');
+    var $clickedElement = $(clickedElement);
+    var panelBody = $clickedElement.parent().find('div[class^="panel-body"]');
     var fsNameForUrl = encodeURIComponent(fsName);
     var url = "/page/instructorFeedbackResponseCommentsLoad?user=" + user + "&courseid=" + courseId + "&fsname=" + fsNameForUrl + "&fsindex=" + fsIndx;
-    $(sender).find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
-    panelBody.load(url, function( response, status, xhr ) {
-        if (status == "success") {
-            panelBody.removeClass('hidden');
-            updateBadgeForPendingComments(panelBody.children(":first").text());
-            panelBody.children(":first").remove();
-            registerResponseCommentsEvent();
-            registerCheckboxEventForVisibilityOptions();
-            enableHoverToDisplayEditOptions();
-      } else {
-          panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
-          panelBody.removeClass('hidden');
-      }
-      $(sender).find('div[class^="placeholder-img-loading"]').html("");
-    });
+    
+    if (!$clickedElement.hasClass("loaded")) {
+        $clickedElement.find('div[class^="placeholder-img-loading"]').html("<img src='/images/ajax-loader.gif'/>");
+        
+        panelBody.load(url, function( response, status, xhr ) {
+            if (status === "success") {
+                updateBadgeForPendingComments(panelBody.children(":first").text());
+                panelBody.children(":first").remove();
+                registerResponseCommentsEvent();
+                registerCheckboxEventForVisibilityOptions();
+                enableHoverToDisplayEditOptions();
+            } else {
+                panelBody.find('div[class^="placeholder-error-msg"]').removeClass('hidden');
+            }
+            
+            //clearQueue to clear the animation queue to prevent animation build up
+            $clickedElement.find('div[class^="placeholder-img-loading"]').html("");
+            $clickedElement.addClass("loaded");
+            $clickedElement.siblings(".collapse").clearQueue().collapse("toggle");
+            toggleChevron(clickedElement);
+        });
+    } else {
+        $clickedElement.siblings(".collapse").clearQueue().collapse("toggle");
+        toggleChevron(clickedElement);
+    }
+}
+
+// sets the chevron of a panel from up to down or from down to up depending on its current state
+// clickedElement must be at least the parent of the chevron
```

Comment:
Is this the recommended formatting for javascript comments?
May be our lead for `t.JS` @bamboo3250 can weigh in?